wallet/src/ui/styles/global.scss

```diff
@@ -34,6 +34,15 @@ body {
     box-sizing: border-box;
 }
 
+::-webkit-scrollbar {
```

Comment:
This makes the scrollbar UX for macos significantly worse (basically unusable outside of a visual indicator). We had a discussion about these scrollbars after implementing them in explorer, and the takeaway was that for now we'll use the platform native scrollbar until we have time to design and implement one with good UX (e.g. Slack, which implement macos-like scrollbars cross-platform with good UX). I imagine that decision also should extend here.

Reply:
Ok.